types/fetch/index.d.ts

```diff
@@ -0,0 +1,111 @@
+declare function fetch(
```

Comment:
We shouldn't need declarations for `fetch` and related types here, we'll get that from `apollo-server-env`.

Reply:
removed